boudicca.events/eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/FemaleCoderCollector.kt

```diff
@@ -0,0 +1,85 @@
+package events.boudicca.eventcollector.collectors
+
+import base.boudicca.SemanticKeys
+import base.boudicca.api.eventcollector.Fetcher
+import base.boudicca.api.eventcollector.TwoStepEventCollector
+import base.boudicca.model.Event
+import base.boudicca.model.EventCategory
+import org.jsoup.Jsoup
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+import java.util.*
+
+class FemaleCoderCollector : TwoStepEventCollector<String>("femalecoder") {
+
+  private val fetcher = Fetcher()
+  private val baseUrl = "https://female-coders.at/"
+
+  override fun getAllUnparsedEvents(): List<String> {
+    val eventUrls = mutableListOf<String>()
+    val document = Jsoup.parse(fetcher.fetchUrl(baseUrl))
+
+    document.select("div.et_pb_section_3 .event_details")
```

Comment:
it is fine as it is, but you could rewrite this forEach loop with a .map operation

then it would look something like

```
val eventUrls = document.select("div.et_pb_section_3 .event_details")
      .map {
        it.select("a").first().attr("href")
      }
```